### DISCUSSION TITLE
Need Help - struggling to verify student status

### DISCUSSION THREAD

**Discussion by aryanpanjwani** [community/community#55258] [GitHub Education] [answered]
### Hi everyone,

Hello, I am a senior at UT Austin trying to verify my status as a student. I have my student ID, but there is no date on it, so that wouldn't suffice. I also don't have any letters or documents with dates on it. I have my student email with a edu extension on my account along with my primary personal email. I don't seem to have any other way to verify it. If there is anything else I can do, or if someone also goes to UT and has a solution of what document they submitted, please let me know. Thanks.

**Comment by jge162** [comment, marked as answer]
I do not go to UT but I used a snippet of my transcript showing term I was registered in (most current) and was approved. Have you tried that yet? @aryanpanjwani

**Reply by aryanpanjwani** [reply]
Yeah, I submitted that on a whim right when I got your reply and it worked. I wasn't sure if it would since it was most recent semester, but that does seem to be a solution. Thanks

**Reply by jge162** [reply]
Noice.

**Reply by amilog** [reply]
I am a second-year computer engineering student at Azerbaijan University. I have been trying to verify my student status for a while, but GitHub does not find the documents I have submitted to be sufficient. What else can I do with the documents provided by my university?

**Reply by Irasil** [reply]
I have the same problem, the verification with the student ID has now been rejected 22 times. I don't know what else I can do. It's really annoying.
